swagger.json

```diff
@@ -5954,6 +5984,55 @@
                 "$ref": "#/definitions/DriverForCreate"
             }
         },
+        "createDvirParam": {
+            "name": "createDvirParam",
+            "description": "DVIR creation body",
+            "required": true,
+            "in": "body",
+            "schema": {
+                "type": "object",
+                "required": [
+                    "vehicleIsSafe"
+                ],
+                "properties": {
+                    "vehicleIsSafe" : {
+                        "type": "boolean",
+                        "description": "Whether or not this vehicle or trailer is safe to drive.",
+                        "example": true
+                    },
+                    "vehicleId" : {
+                        "type": "integer",
+                        "description": "Id of vehicle being inspected.",
+                        "example": 10
+                    },
+                    "trailerId" : {
+                        "type": "integer",
+                        "description": "Id of trailer being inspected.",
+                        "example": 11
+                    },
+                    "previousDefectsCorrected": {
+                        "type": "boolean",
+                        "description": "Whether any previous defects were corrected.",
+                        "example": true
+                    },
+                    "previousDefectsIgnored": {
+                        "type": "boolean",
+                        "description": "Whether any previous defects were ignored.",
+                        "example": false
+                    },
+                    "odometerMiles": {
+                        "type": "integer",
```

Comment:
Is this an integer or a floating point value?

Reply:
All our other odometer numbers are integers, and the DVIR list endpoint returns odometers as integers - I'd like to keep that consistent.